src/main/java/io/stargate/sgv2/jsonapi/StargateJsonApi.java

```diff
@@ -673,6 +673,83 @@
                         }
                       }
                       """),
+              @ExampleObject(
+                  name = "findVectorProviders",
+                  summary = "`findVectorProviders` command result",
+                  value =
+                      """
+                        {
+                            "status": {
```

Comment:
"status"? That seems like an odd choice for JSON key here. Is that actually used or left from copying another example?

Reply:
The example is the actual return. I use this by following the [spec](https://docs.google.com/document/d/1XkCLUI1YgMuqqUU0KYMfyhgBgRmdC4aSVQXZrodV5T0/edit#heading=h.qitn2whv4gd3) example. Do we need to change it to a better name?